plugins/event_engine_plugin/event_engine_plugin.cpp

```diff
@@ -214,37 +215,31 @@ void event_engine_plugin_impl::send_genesis_file(const bfs::path& genesis_file)
     using cyberway::genesis::ee_table_header;
 
     std::cout << "Reading event engine genesis data from " << genesis_file << "..." << std::endl;
-    bfs::ifstream in(genesis_file);
-    ee_genesis_header h{"", 0};
+    boost::iostreams::mapped_file mfile;
+    mfile.open(genesis_file, boost::iostreams::mapped_file::readonly);
+    EOS_ASSERT(mfile.is_open(), ee_extract_genesis_exception, "File not opened");
 
-    in.read((char*)&h, sizeof(h));
+    ee_genesis_header &h = *(ee_genesis_header*)mfile.const_data();
```

Comment:
`const auto`?

Reply:
`const` - Ok. but in this place I need guarantees that this is exactly this type, so without `auto`.

Reply:
@s-medvedev C++ is not interpreting :) There will be `ee_genesis_header` anyway.